plugins/with-postgres/package.json

```diff
@@ -22,13 +22,13 @@
     "@build-tracker/api-errors": "^1.0.0-beta.12",
     "@build-tracker/build": "^1.0.0-beta.12",
     "@build-tracker/server": "^1.0.0-beta.13",
+    "@types/pg": "^7.4.13",
```

Comment:
I was getting type errors when using the `withPostgres` function. The issue is that the declaration files point to missing dependencies, thus we must install these as normal dependencies instead of dev ones. https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies

Reply:
Does it have to be a production dependency?

Reply:
That just seems... stupid.

Reply:
I'm afraid so, otherwise consumers wouldn't install it and they would get type errors for a missing dependency. IIUC, this is common practice.

- https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies
- https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies
- https://github.com/microsoft/types-publisher/issues/81

Reply:
🤦‍♂